src/mnemonics/electrum-words.cpp

```diff
@@ -371,6 +371,59 @@ namespace crypto
       return true;
     }
 
+    std::string get_invalid_word(const epee::wipeable_string &words)
+    {
+      // If there's a new language added, add an instance of it here.
```

Comment:
It might be worth the additional diff to find where else language lists are defined and leave a comment directing devs here for language-related work